WooCommerce/Classes/ViewRelated/Blaze/ProductSelector+Blaze.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+internal extension ProductSelectorView.Configuration {
```

Comment:
Nit: `internal` can be removed because it's the default scope for a type.

Reply:
👍🏼 Handled in https://github.com/woocommerce/woocommerce-ios/pull/11625/commits/864f2b31551b83acef03aa79748c637d5d32e244